client/dashboard/section.js

```diff
@@ -0,0 +1,58 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+export default class StorePerformance extends Component {
+	constructor( props ) {
+		super( props );
+		const { title } = props;
+
+		this.state = {
+			titleInput: title || __( 'Store Performance', 'woocommerce-admin' ),
```

Comment:
You can remove the `||` part of this statement. `onMenuToggle` should simply not `setState` if string no string is entered. This is what Timmy noticed in his comment.